servers/rendering/rasterizer.h

```diff
@@ -815,7 +815,8 @@ class RasterizerCanvas {
 		CANVAS_RECT_FLIP_H = 4,
 		CANVAS_RECT_FLIP_V = 8,
 		CANVAS_RECT_TRANSPOSE = 16,
-		CANVAS_RECT_CLIP_UV = 32
+		CANVAS_RECT_CLIP_UV = 32,
+		CANVAS_RECT_IS_GROUP = 64
```

Comment:
Note: If you take the habit to add a trailing comma to enums, you avoid this situation where a needless change is done on the previous last value.